common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -30,14 +32,43 @@ class EnrollmentTest(ModuleStoreTestCase, APITestCase):
     USERNAME = "Bob"
     EMAIL = "bob@example.com"
     PASSWORD = "edx"
+    API_KEY = "i am a key"
 
     def setUp(self):
         """ Create a course and user, then log in. """
         super(EnrollmentTest, self).setUp()
         self.course = CourseFactory.create()
         self.user = UserFactory.create(username=self.USERNAME, email=self.EMAIL, password=self.PASSWORD)
+        self.other_user = UserFactory.create()
         self.client.login(username=self.USERNAME, password=self.PASSWORD)
 
+    def _create_enrollment(self, course_id=None, username=None, expected_status=status.HTTP_200_OK, email_opt_in=None, as_server=False):
+        """Enroll in the course and verify the URL we are sent to. """
```

Comment:
It was squashed between two tests near the bottom. I tend to like helper methods at the top, but I'm not super wedded to that idea.